Source/AliveLibAE/Renderer/DirectX9Renderer.cpp

```diff
@@ -20,6 +20,8 @@ void DirectX9Renderer::Destroy()
 
 bool DirectX9Renderer::Create(TWindowHandleType window)
 {
+    rendererType = Renderers::DirectX9;
```

Comment:
This isn't used anywhere

Reply:
Use polymorphic calls to abstract it out - e.g Create is a virtual call so DirectX9Renderer::Create know its the directx impl. If something outside of the object using the base type needs to know this then its probably leaking implementation details.

But its not even used currently so its dead code anyway.

Reply:
alright. done.